Gemfile

```diff
@@ -246,15 +246,17 @@ end
 # To develop a gem locally and override its source to a checked out repo
 #   you can use this helper method in Gemfile.dev.rb e.g.
 #
-# override_gem 'manageiq-ui-classic', :path => File.expand_path("../manageiq-ui-classic", __dir__)
+# override_gem 'manageiq-ui-classic', :path => "../manageiq-ui-classic"
 #
-def override_gem(name, *args)
+def override_gem(name, args)
   if dependencies.any?
     raise "Trying to override unknown gem #{name}" unless (dependency = dependencies.find { |d| d.name == name })
     dependencies.delete(dependency)
 
+    args[:path] = File.expand_path(args[:path], __dir__) unless args[:path].nil?
```

Comment:
Prefer the positive case `if args[:path]`

Reply:
Also, I think we need to `expand_path` from the path of the `calling_file` below rather than `__dir__` to ensure that relative paths work.  Currently if I use `../../manageiq-ui-classic` (next to manageiq) and specify that in bundler.d/Gemfile.dev.rb, it puts me in the wrong place since this is all evaluated from Gemfile, not bundler.d/Gemfile.dev.rb

Reply:
Done, getting the calling dir with `File.dirname(calling_file)`